src/components/FeatureViews/Registration/RegistrationCart/RegistrationCartContainer.js

```diff
@@ -87,21 +86,36 @@ export default function RegistrationCartContainer() {
 	const studentIds = (Object.keys(registrationCartState).length > 0 && Object.keys(registrationCartState)) ||
 		currentParent.studentList;
 	// create list of courses to fetch
-	const courseIds = [].concat.apply([], Object.values(registrationCartState))
-		.map(({course}) => course.id);
-	const {data, loading} = useQuery(GET_STUDENT_INFOS, {variables: {userIds: studentIds}});
-	const coursesResponse = useQuery(GET_COURSES_TO_REGISTER, {variables: {courseIds: courseIds}});
+	const courseIds = Object.values(registrationCartState).filter(reg => reg).length > 0 &&
```

Comment:
This condition is not really necessary, as the following value defaults to an empty array if there are no non-empty registrations. However, you will have to check for an array of length 0 instead of `false` when seeing if any IDs are present.

Reply:
there's an extra submitStatus key that gets generated into the registrationCartState when you pass it through redux. I couldn't find where to get rid of it, so i'm just filtering it out since it's value is null